teamspeak.js

```diff
@@ -37,16 +37,22 @@ class Teamspeak {
      * @return Promise{boolean}
      */
     async messageUser(tsUid, message){
-        //We first need to get the TeamspeakClient
-        let tsClientList = await this.ts3.clientList({client_type: 0, client_unique_identifier: tsUid});
+        try {
+            //We first need to get the TeamspeakClient
+            let tsClientList = await this.ts3.clientList({client_type: 0, client_unique_identifier: tsUid});
 
-        //Check if tsClientList contains at least one client
-        if(tsClientList.length > 0)
-        {
-            let tsClient = tsClientList[0];
-            tsClient.message(message)
-                .then(() => {return true;})
-                .catch(() => {return false;});
+            //Check if tsClientList contains at least one client
+            if(tsClientList.length > 0)
+            {
+                let tsClient = tsClientList[0];
+                await tsClient.message(message);
+                return true
+            }
+                
+        } catch (e) {
+            logger.error(`Error while trying to message user with ${tsUid}`)
+            logger.error(e)
+            return false
```

Comment:
Would'nt it be better to move the return statement inside the catch clause outside? Because if tsClientList.length = 0 then no message would be sent and no return value would be given back?
PS: I am new to node.js, so please correct me, if I am wrong

Reply:
I was not really sure what it actually should return since this function never gets actually used

Reply:
Yeah, I think I wrote it because I thought it would be useful one day. Theoretically we could just remove it completely...